### ISSUE TITLE
Removing USB dongle causes full Chrome restart

### ISSUE THREAD

**Issue by jsquyres** [beagleterm/beagle-term#33]
I don't know if this is a BeagleTerm or ChromeOS issue, but I found that if I have a BeagleTerm window open / connected to my USB-to-serial dongle, and if I physically remove the USB dongle, the entire Chrome browser (on my Chromebook) will reset, and I'll get the "Chrome didn't shut down properly; click here to restore all tabs" button.

I have ChromeOS 37.0.2062.120 (64 bit) on and HP Chromebook.
Platform 5978.98.0 (official build) platform falco
Firmware Google_Falco.4389.78.0

I have BeagleTerm v1.3.

**Comment by RandomlyKnighted** [comment]
This has been fixed by Chrome. I booted up my Cisco 1811 Router and so that I could see that there was a connection between BeagleTerm and another device. I then removed my USB to Serial adapter and the app remained open. 

There is; however, one downside to this good news. When the dongle is removed there is currently no notification to the user that the USB adapter has been removed. The app does not recognize any type of input from the user at this point.
